tools/distrib/install_all_python_modules.sh

```diff
@@ -13,45 +13,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO: Integrate this into CI to avoid bitrot.
+set -ex
 
-echo "It's recommended that you run this script from a virtual environment."
+BASEDIR=$(dirname "$0")/../..
+PACKAGES="grpcio_channelz  grpcio_csds  grpcio_admin grpcio_health_checking  grpcio_reflection  grpcio_status  grpcio_testing grpcio_csm_observability grpcio_tests"
+
+# Change to grpc repo root
+cd "$BASEDIR";
 
+# unit-tests setup starts from here
 function maybe_run_command () {
-  if python setup.py --help-commands | grep "$1" &>/dev/null; then
-    python setup.py "$1";
+  if python3 setup.py --help-commands | grep "$1" &>/dev/null; then
+    python3 setup.py "$1";
   fi
 }
 
-set -e
+python3 -m pip install --upgrade "cython==3.0.0";
```

Comment:
We should probably use `cython<4.0.0rc1`

Reply:
updated